.github/workflows/e2e.yaml

```diff
@@ -92,7 +92,10 @@ jobs:
           echo "Using cluster name \"$CLUSTER_NAME\""
           echo CLUSTER_NAME=$CLUSTER_NAME >> $GITHUB_ENV
       - name: create eks cluster '${{ env.CLUSTER_NAME }}'
-        uses: ./.github/actions/e2e/create-cluster
+        # Revert back to local action if either occurs:
+        # 1) Get a Dockerfile that reduces the number of local steps that we run 
+        # 2) Github fixes the issue that is causing this failure
+        uses: aws/karpenter/.github/actions/e2e/create-cluster@main
```

Comment:
Can this just reference the local path? I don't think we were having issue in the post steps for this workflow here